src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.img__link {
+  display: block;
+  height: 40px;
+}
+
+header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
```

Comment:
You are not supposed to set height for header. Instead let links to dictate the height of it ;)